dashboard/modules/node/node_head.py

```diff
@@ -242,7 +244,9 @@ async def get_all_nodes(self, req) -> aiohttp.web.Response:
         if view == "summary":
             all_node_summary = await DataOrganizer.get_all_node_summary()
             return dashboard_optional_utils.rest_response(
-                success=True, message="Node summary fetched.", summary=all_node_summary
+                success=True,
```

Comment:
The recommendation was to return logical resources with the nodes endpoint, so a separate API call is not needed in the clusters page

Reply:
@alanwguo 
For my current implementation, we still need to call a separate API, api/cluster_status, on the cluster page.

The expectation is that on the cluster page, we should only need to call nodes/ API. Therefore, I need to include all logical resource information in the nodes/ API.